scripts/do-release.sh

```diff
@@ -250,6 +263,52 @@ function post_reviews_comment() {
     dryrun gh pr review "${pr_url}" --comment --body "${comment}"
 }
 
+function get_bundle_pr_body() {
+   echo "Release Submariner v$1"
+   curl -s "${BUNDLE_PR_TEMPLATE}" | sed -E "s/\[ \]/\[x\]/g; 0,/Is operator/d"
+}
+
+function release_bundle() {
+    local bundle_version="${release['version']#v}"
+    local bundle_from_version="${release['bundle.from_version']:-0.0.0}"
+    local bundle_channel="${release['bundle.channel']:-$(echo alpha-"${bundle_version}" | cut -d'.' -f1,2)}"
+    local pr_body
+    pr_body=$(get_bundle_pr_body "${bundle_version}")
+
+    (
+        project=submariner-operator
+        pushd projects/${project}
+        dapper_in_dapper
+
+        make packagemanifests \
+            VERSION="${bundle_version}" \
+            FROM_VERSION="${bundle_from_version}" \
+            CHANNEL="${bundle_channel}"
+
+        # Running dapper_in_dapper changes the Dockerfile, make sure to reset it.
+        _git checkout HEAD Dockerfile.dapper
```

Comment:
```suggestion
        git checkout HEAD Dockerfile.dapper || :
```

Otherwise it will fail on newer versions.
Also `_git` runs outside the project dir, you want the plain old `git`